File: wiki_starwars/lib/model/specie_model.dart
class SpecieModel {
  String _name;

  SpecieModel({String name});

  String get name => _name;
  set name(String name) => _name = name;

  SpecieModel.toMap(Map<String, dynamic> json) {
    _name = json['name'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['name'] = this._name;
    return data;
  }
}

File: app/src/main/java/com/siedg/desafio_android/data/model/SpecieModel.kt
package com.siedg.desafio_android.data.model

import com.google.gson.annotations.SerializedName

class SpecieModel(
    val averageHeight: String,
    val averageLifespan: String,
    val classification: String,
    val created: String,
    val designation: String,
    val edited: String,
    val eyeColor: String,
    val hairColor: String,
    val homeworld: String,
    val language: String,
    val name: String,
    val people: List<String>,
    val films: List<String>,
    val skinColor: String,
    val url: String
)